src/storage.rs

```diff
@@ -138,13 +137,12 @@ pub fn can_reach_datastore(datastore: &DataStore) -> bool {
             request_payer: None,
         })
         .sync()
-    {
-        Ok(_) => true,
-        Err(e) => {
+        .map_err(|e| {
             info!("Cannot access bucket: {}", e);
-            false
-        }
-    }
+            e
+        })
+        .map(|_| true)
+        .unwrap_or(false)
```

Comment:
will it always unwrap true?

Reply:
yeah `unwrap_or` is if the result is an ok, return that otherwise return the given value.